hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/event/AsyncDispatcher.java

```diff
@@ -189,10 +189,24 @@ protected void serviceInit(Configuration conf) throws Exception{
     ThreadFactory threadFactory = new ThreadFactoryBuilder()
         .setNameFormat("PrintEventDetailsThread #%d")
         .build();
+
```

Comment:
nit: avoid this

Reply:
Thank you very much for helping to review the code, I will modify the code.